src/test/java/org/jabref/logic/util/BracketedPatternTest.java

```diff
@@ -59,13 +59,30 @@ public void nullDatabaseExpansionTest() {
         BracketedPattern pattern = new BracketedPattern("[year]_[auth]_[firstpage]");
         assertEquals("2017_Kitsune_213", pattern.expand(bibentry, another_database));
     }
+    
+    @Test
+    public void pureauthTest() {
+        BibDatabase another_database = new BibDatabase();
```

Comment:
Please don't use underscores, but camel case, i.e., `anotherDatabase`. (here and in the test below)

Reply:
Thanks for your reply. I will change it.